src/screens/Map/styles.js

```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
You also may: 
```
container: {
  ...StyleSheet.absoluteFillObject,
  justifyContent: 'flex-end',
   alignItems: 'center',
}
```
[https://facebook.github.io/react-native/docs/stylesheet#absolutefillobject](https://facebook.github.io/react-native/docs/stylesheet#absolutefillobject)